ios/BetterRail.xcodeproj/project.pbxproj

```diff
@@ -1339,7 +1347,7 @@
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 5.0;
 				TARGETED_DEVICE_FAMILY = 4;
-				WATCHOS_DEPLOYMENT_TARGET = 7.1;
+				WATCHOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
Why is that?

Reply:
`.navigationBarTitleDisplayMode(.inline)` requires watchOS 8+